ui/vic-ui-h5c/vic/src/vic-app/src/app/summary-portlet/summary-portlet.component.ts

```diff
@@ -77,7 +77,12 @@ export class VicSummaryPortletComponent implements
 
     ngAfterViewInit() {
         setTimeout(() => {
-            this.service.fetchVmInfo(VM_PROPERTIES_TO_EXTRACT, this.stubType);
+            // set up objectId in data property service
+            this.route.params.subscribe((params: any) => {
+                this.service.setObjectId(params.id);
+                this.service.fetchVmInfo(VM_PROPERTIES_TO_EXTRACT, this.stubType);
+                console.log(`objectId set to ${params.id}`);
```

Comment:
All `console.log` statements get removed during the production build process. Those are quite useful when testing with the dev build mode which doesn't remove them.